core/src/player.rs

```diff
@@ -3,7 +3,7 @@ use crate::avm1::debug::VariableDumper;
 use crate::avm1::globals::system::SystemProperties;
 use crate::avm1::object::Object;
 use crate::avm1::property::Attribute;
-use crate::avm1::{Avm1, AvmString, ScriptObject, TObject, Timers, Value};
+use crate::avm1::{Avm1, AvmString as Avm1String, ScriptObject, TObject, Timers, Value};
```

Comment:
Is there a reason for this. AFAIK AVM1 and AVM2 both use the same string representation and this PR doesn't appear to split them out...

Reply:
Hm, you're right; I think I did this because `AvmString` is in the `ruffle_core::avm1` module, but it doesn't actually makes sense ^^.

Maybe `AvmString` should be moved in `ruffle_core::string` in another PR?